src/sendGCM.js

```diff
@@ -105,12 +105,12 @@ const sendGCM = (regIds, data, settings) => {
   }
 
   const message = new gcm.Message({
-    // See https://developers.google.com/cloud-messaging/http-server-ref#table5
+    // See https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#AndroidConfig
     collapseKey: data.collapseKey,
     priority: data.priority === 'normal' ? data.priority : 'high',
     contentAvailable: data.contentAvailable || false,
     delayWhileIdle: data.delayWhileIdle || false,
-    timeToLive:
+    ttl:
```

Comment:
we do not construct FCM messages ourselves but use node-gcm. the field name is timeToLive, see https://github.com/ToothlessGear/node-gcm#usage
If the underlying mapping must be changed for FCM compatibility, please request a fix in the node-gcm lib